tests/gcloud-testing-core/src/main/java/com/google/cloud/runtime/jetty/testing/RemoteLog.java

```diff
@@ -0,0 +1,224 @@
+package com.google.cloud.runtime.jetty.testing;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.junit.Assert.assertThat;
+
+import com.google.cloud.MonitoredResource;
+import com.google.cloud.Page;
+import com.google.cloud.logging.LogEntry;
+import com.google.cloud.logging.Logging;
+import com.google.cloud.logging.LoggingOptions;
+import com.google.cloud.logging.Payload;
+import com.google.gson.Gson;
+import com.google.gson.stream.JsonReader;
+
+import org.eclipse.jetty.toolchain.test.MavenTestingUtils;
+
+import org.yaml.snakeyaml.Yaml;
+import org.yaml.snakeyaml.parser.ParserException;
+import org.yaml.snakeyaml.scanner.ScannerException;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+import java.util.ListIterator;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+
+public class RemoteLog {
```

Comment:
Same here, it's not used as part of this PR. Let's remove it for now, and add it in as part of a separate PR that uses it.